contracts/ETO/ETOCommitment.sol

```diff
@@ -835,17 +843,20 @@ contract ETOCommitment is
             isEligible,
             minTicketEurUlps,
             maxTicketEurUlps,
-            equityTokenInt,
-            fixedSlotEquityTokenInt
+            equityTokenAmount,
+            fixedSlotEquityTokenAmount
         ) = ETO_TERMS.calculateContribution(investor, _totalEquivEurUlps, ticket.equivEurUlps, equivEurUlps, applyDiscounts);
         // kick out on KYC
         require(isEligible, "NF_ETO_INV_NOT_ELIGIBLE");
         // kick on minimum ticket and you must buy at least one token!
-        require(equivEurUlps + ticket.equivEurUlps >= minTicketEurUlps && equityTokenInt > 0, "NF_ETO_MIN_TICKET");
+        require(
+            equivEurUlps + ticket.equivEurUlps >= minTicketEurUlps &&
+            equityTokenAmount >= EQUITY_TOKEN_POWER &&
+            equityTokenAmount > 0, "NF_ETO_MIN_TICKET");
```

Comment:
EQUITY_TOKEN_POWER will always be at least 1, so the second check is not needed

Reply:
true, this is what you've get when you fix test cases without looking at the whole condition :)